conjur/cli.py

```diff
@@ -20,6 +19,9 @@
 # Internals
 from conjur.argparse_wrapper import ArgparseWrapper
 from conjur.client import Client
+from conjur.init_logic import InitLogic
+from conjur.init_controller import InitController
```

Comment:
This separation of module code isn't in line w/ Python standards. This should be:
```
conjur/init/__init__.py
conjur/init/foo.py
conjur/init/bar.py
```
all imported from `conjur/init/__init__.py` to hide the details, kind of like we do here: https://github.com/cyberark/conjur-api-python3/blob/master/conjur/__init__.py

Reply:
Are you suggesting to create a new init folder to hold all init logic? or create a new __init__.py for init and have all imports and init related objects initialized there?

Reply:
> Are you suggesting to create a new init folder to hold all init logic?

exactly. you have too much logic that should span multiple files but it all is related to "init"/configuration

Reply:
fixed